app/javascript/components/OrganizationPage.tsx

```diff
@@ -73,27 +110,50 @@ const OrganizationPage = () => {
         </div>
       </div>
 
-      {/* <div className="card mb-9">
-        <h2 className="mb-3">Plan</h2>
-        <div className="mb-6">
-          Learn more about or compare plans by visiting our{' '}
-          <a className="text-link" href="/pricing" target="_blank">
-            pricing page
-          </a>
-          .
-        </div>
-        <Tooltip content="Under development">
-          <Button color="light">Add card</Button>
-        </Tooltip>
-      </div> */}
-
-      {/* <div className="card mb-9">
-        <h2>Payment Method</h2>
-        <div className="mb-6">No card on file.</div>
-        <Tooltip content="Under development">
-          <Button color="light">Add card</Button>
-        </Tooltip>
-      </div> */}
+      <div className="card mb-9">
+        <h2 className="flex justify-between">Subscription</h2>
+
+        {!hasActiveSubscription && (
+          <>
+            <div className="mb-6">
+              Learn more about or compare plans by visiting our{' '}
+              <a className="text-link" href="/pricing" target="_blank">
+                pricing page
+              </a>
+              .
+            </div>
+            <div className="flex gap-3">
+              {getPlans().map((plan) => (
+                <Card key={plan.priceId} className="w-1/3">
+                  <div className="text-xl font-bold">{plan.name}</div>
+                  <p>{plan.conditions}</p>
+                  <form action="/user/subscriptions.json" method="POST">
+                    <input type="hidden" name="authenticity_token" value={csrfToken || ''} />
+                    <input type="hidden" name="organization_id" value={organizationId} />
+                    <input type="hidden" name="price_id" value={plan.priceId} />
+                    <Button type="submit" fullSized>
+                      Pay ${plan.price / 100} per {plan.interval}
+                    </Button>
+                  </form>
+                </Card>
+              ))}
+            </div>
+          </>
+        )}
```

Comment:
Why not showing the Stripe billing management page ? 
Ok you done it now, but, I think this is the most simple way to manage stripe, because it displays the plan, the customer can choose or change the plan, and you don't care about building this view
<img width="1463" alt="Capture d’écran 2023-10-13 à 09 55 12" src="https://github.com/cherrypush/cherrypush.com/assets/10012140/40f25420-c6e2-4991-b8ee-3519a113b829">
WDYT ? :)

Reply:
Nice. I didn't know about it. 
Do you know if we can set the billing management page to a specific customer id?
I'll check the docs anyways :)

Reply:
From what I see it's the same problem I faced with the payment links where I can't set a specific customer 😞 

![image](https://github.com/cherrypush/cherrypush.com/assets/1740848/5491cb27-8b06-4933-a768-d0cc357581a3)

Reply:
The thing is that our users are actually organizations. 
So a single user can be responsible for multiple organizations. 
Or multiple organizations might use the same billing email. 

Anyways, the point is that I need a strong link between the checkout session and the customer, which is what I achieved by manually creating the Session on our back-end, then redirecting the user to the stripe session page. 

Then the user is redirected back to Cherry, and I fetch the stripe API for their active subscriptions :) 

It's quite simple actually. But sure, it could be simpler if the subscription was at the account level.